test/e2e/lib/flows/login-flow.js

```diff
@@ -95,6 +96,13 @@ export default class LoginFlow {
 		}
 
 		await loginPage.login( this.account.email || this.account.username, this.account.password );
+
+		if ( process.env.FLAGS === 'nav-unification' ) {
+			// Makes sure that the nav-unification welcome modal will be dismissed.
+			const guideComponent = new GuideComponent( this.driver, 1000 );
```

Comment:
What does `1000` do? The arg signature for the AsyncBaseContainer is

```
driver,
expectedElementSelector,
url = null,	
waitMS = config.get( 'explicitWaitMS' )
```

Reply:
ahh... this is the `waitOverride` https://github.com/Automattic/wp-calypso/blob/ae3be17749205375effb3ebb981cfdbd324f3071/test/e2e/lib/components/guide-component.js#L18 that I have put in the wrong place.. Fixed here 73d396e, very good catch !